README.md

```diff
@@ -40,11 +40,16 @@ These dependencies must be present before building:
  - `libwnck-3-dev`
 
 
-Use the following command to install the dependencies:
+Use the following command to install the dependencies on Elementary:
 ```shell
 sudo apt install elementary-sdk libgtop2-dev libwnck-3-dev
 ```
  
+Use the following command to install the dependencies on other Debian-based systems:
+```shell
+apt install meson valac appstream-util cmake libgranite-dev libgtop2-dev libwnck-3-dev
```

Comment:
You're right. The `cmake` warning I got was related to not having `valac` installed. Once that's installed, the `cmake` warning goes away as well.

Reply:
I don't know if you actually need cmake for the app itself - Optimizer uses meson as the build system. Have you tried this?